README.md

```diff
@@ -59,61 +65,77 @@ class { 'nsclient':
 }
 ```
 
-##Usage
+## Reference
+
+### Classes and Defined Types
 
-###Classes and Defined Types
+#### Class: `nsclient`
 
-####Class: `nsclient`
-One of the primary classes of the nsclient module. This class will install the nsclient binaries
+One of the primary classes of the nsclient module. This class will install the
+nsclient binaries
 
 **Parameters within `nsclient`:**
-#####`allowed_hosts`
-Array of hosts that your client can communicate with. You can use netmasks (/ syntax) or * to create ranges.
 
-#####`service_state`
+##### `allowed_hosts`
+
+Array of hosts that your client can communicate with. You can use netmasks
+(/ syntax) or * to create ranges.
+
+##### `service_state`
+
 Whether you want to nsclient service to start up. Defaults to running
 
-#####`service_enable`
+##### `service_enable`
+
 Whether you want to nsclient service to start up at boot. Defaults to true
 
-#####`package_source_location`
-This is the default site to download your package from (e.g. http://files.nsclient.org/stable)
+##### `package_source_location`
+
+This is the default site to download your package from (e.g. <http://files.nsclient.org/stable>)
+
+##### `package_source`
 
-#####`package_source`
 This is the source name of the package to be downloaded
 
-#####`package_name`
+##### `package_name`
+
 This is name of the package to download (e.g. NSCP-0.4.1.101-x64.msi)
 
-#####`download_destination`
-This is the folder to where we need to download the NSCP Installer. Package cannot take a remote file source
-Because of Windows, we need to set this to be a top level directory (e.g. c:\\temp) or we would need to
-recursively check the file path.
+##### `download_destination`
+
+This is the folder to where we need to download the NSCP Installer. Package
+cannot take a remote file source Because of Windows, we need to set this to be
```

Comment:
Missing punctuation? should be `...take a remote file source.  Because of Windows, ...`